packages/kit/src/module/install.ts

```diff
@@ -61,18 +61,28 @@ export async function loadNuxtModuleInstance (nuxtModule: string | NuxtModule, n
   let buildTimeModuleMeta: ModuleMeta = {}
   // Import if input is string
   if (typeof nuxtModule === 'string') {
-    const src = await resolvePath(nuxtModule)
-    try {
+    const paths = [join(nuxtModule, 'module'), nuxtModule]
```

Comment:
It would be nice if we also add `/nuxt` prefix for opt-in modules support from libraries.

Reply:
```suggestion
    const paths = [join(nuxtModule, 'nuxt'), join(nuxtModule, 'module'), nuxtModule]
```